Formula/netmask.rb

```diff
@@ -0,0 +1,20 @@
+class Netmask < Formula
+  desc "IP address netmask generation utility"
+  homepage "https://github.com/tlby/netmask/blob/master/README"
+  url "https://github.com/tlby/netmask/archive/refs/tags/v2.4.4.tar.gz"
+  sha256 "7e4801029a1db868cfb98661bcfdf2152e49d436d41f8748f124d1f4a3409d83"
+  license "GPL-2.0-only"
+
+  depends_on "automake" => :install
+
+  def install
+    system "./autogen"
```

Comment:
Does the `configure` it ships not work on macOS?

Reply:
Yeah it is actually a `configure.ac` file so it requires automake to install it. `autogen` is the first step